### PR TITLE
Support --opa-auth-token-file

### PR DESCRIPTION
**Pull request by kfox1111**
This PR adds support for reading the opa auth token from
a file so it isn't leaked out in a process list.

### CODE REVIEW

**Comment by repenno** [comment]
Hello @kfox1111 

I second what @pwmorreale said. What does the file looks like? For example, it seems if I add more than one token it would be silently discarded based on: (strings.Split(string(file), "\n")[0])

thanks,

**Comment by kfox1111** [comment]
Ok. I'll add a fix to make it mutually exclusive.

The only documentation I can find is in docs/admission-control-secure.md

But it is not easy to document the usage of both. Should it be covering --opa-auth-token or --opa-auth-token-file? Maybe this decision should be left up to a follow on ps?

**Comment by tsandall** [comment]
@kfox1111 if you want to modify the docs to use the file-based version (and leave out the command line version) that would be fine.

**Comment by kfox1111** [comment]
Thats correct. it reads the token out of the file. if there are multiple lines in the file, it reads the first line...

How do you want that documented? it wont fit in --help, and its not obvious where to document that one feature.

**Comment by kfox1111** [comment]
@tsandall ok, will do. thanks.

File: docs/admission-control-secure.md [hunk 1]
```diff
@@ -109,39 +109,39 @@ spec:
       args:
         - "--replicate-cluster=v1/namespaces"
         - "--replicate=extensions/v1beta1/ingresses"
-        - "--opa-auth-token=kube-mgmt"
+        - "--opa-auth-token-file=/policy/token"
```
Comment (tsandall) [inline docs/admission-control-secure.md]: I think this should be `/policies/token` to match the volumeMount below.

Comment (kfox1111) [reply docs/admission-control-secure.md]: ah. good catch. thanks.

**Comment by tsandall** [comment]
@kfox1111 LGTM. If you squash your changes, I'll merge the PR. Thanks for taking care of this.

**Pull request merged**